cirq/_doc.py

```diff
@@ -24,17 +24,24 @@
 RECORDED_CONST_DOCS: Dict[int, DocProperties] = {}
 
 
-def document(value: Any, doc_string: str):
+def document(value: Any, doc_string: Optional[str] = None):
     """Stores documentation details about the given value.
 
-    This method is used to associate a docstring with global constants.
+    This method is used to associate a docstring with global constants. It is
+    also used to indicate that a private method should be included in the public
+    documentation (e.g. when documenting protocols or arithmetic operations).
 
     The given documentation information is filed under `id(value)` in
     `cirq._doc.RECORDED_CONST_DOCS`.
 
     Args:
         value: The value to associate with documentation information.
-        doc_string: The doc string to associate with the value.
+        doc_string: The doc string to associate with the value. Defaults to the
+            value's __doc__ attribute.
+
+    Returns:
+        The given value.
     """
     docs = DocProperties(doc_string=doc_string)
     RECORDED_CONST_DOCS[id(value)] = docs
+    return value
```

Comment:
Does this fix https://github.com/quantumlib/Cirq/issues/2669?

Reply:
I don't think so. I thought that was already fixed by 0.7

Reply:
It was. I don't know which PR fixed it though. I'll close it.